test-send-retrieve.coffee

```diff
@@ -127,10 +146,14 @@ do ->
       .replace /#{{repoUniqueId}}/g, conf.repoUniqueId
       .replace /#{{firstName}}/g, conf.firstName
       .replace /#{{lastName}}/g, conf.lastName
+      .replace /#{{randomUUID}}/g, -> uuid.v4()
 
-  provideDoc = process fs.readFileSync('pnr-validAphpSampleFullSections.xml').toString()
+  provideDoc = process fs.readFileSync('pnr-minimalIC.xml').toString()
   retrieveDoc = process fs.readFileSync('retrieveDocumentSetb.xml').toString()
+  adt = process fs.readFileSync('adt_a01.er7').toString()
 
-  runTest 'Provide and Register Document Set.b', provideAndRegister, isPnRSuccessful, conf, provideDoc, provideDoc, (success) ->
+  sendRegistyHL7 adt, (success) ->
     if success
```

Comment:
Maybe just add an `else` condition for this with a console log explaining why it failed